OpenRA.Mods.Common/Traits/AffectsShroud.cs

```diff
@@ -34,7 +34,8 @@ public abstract class AffectsShroud : ConditionalTrait<AffectsShroudInfo>, ITick
 		readonly HashSet<PPos> footprint;
 
 		[Sync] CPos cachedLocation;
-		[Sync] bool cachedTraitDisabled;
+		[Sync] WDist cachedRange;
+		[Sync] protected bool CachedTraitDisabled { get; private set; }
```

Comment:
Please cherry pick 7b1567ca785c7652045e9037a50e0d4211d6f604 to be the first commit in this PR.  Otherwise it crashes on startup, which will create a problem when bisecting.